README.md

```diff
@@ -1,24 +1,158 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Catalog Of My Things
 
-Things you may want to cover:
+## _Microverse Rails API and React Final Group Capstone Project_
 
-* Ruby version
+> This is an educational project to create a console app that you to keep a record of different types of things you won: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below.
```

Comment:
- [ ] Please, kindly add full API documentation, if your team includes 3 or more people.  I would suggest you add swagger.